src/main/java/com/exasol/sql/ValueTableRow.java

```diff
@@ -1,8 +1,12 @@
 package com.exasol.sql;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
-import com.exasol.sql.expression.*;
+import com.exasol.sql.expression.DoubleLiteral;
```

Comment:
Please check why `DoubleLiteral` is not in the package `literal`. This looks like an accident.

Reply:
:+1: